lib/slack.rb

```diff
@@ -1,11 +1,74 @@
 #!/usr/bin/env ruby
+require_relative "user"
+require_relative "channel"
+require_relative "workspace"
+require "httparty"
+require "table_print"
+require "dotenv"
+Dotenv.load
 
 def main
   puts "Welcome to the Ada Slack CLI!"
+  puts "The workspace has #{SlackAPI::Channel.list_channels.length} channels and #{SlackAPI::User.list_users.length} users"
+  ask_again = "Select an option: \n1. list users \n2. list channels \n3. select user \n4. select channel \n5. details (select user or channel first)\n6. send message (select user or channel first) \n7. quit"
+  puts ask_again
+  response = gets.chomp.downcase
 
-  # TODO project
-
-  puts "Thank you for using the Ada Slack CLI"
+  until response == "7" || response == "quit"
+    if response == "list users" || response == "1"
```

Comment:
This set of if-else blocks, really looks like it could be broken up into methods.